Ryujinx.Audio/Renderers/SoundIo/SoundIoAudioOut.cs

```diff
@@ -77,12 +69,12 @@ public PlaybackState GetState(int trackId)
         /// <returns>The created track's Track ID</returns>
         public int OpenTrack(int sampleRate, int channels, ReleaseCallback callback)
         {
-            if (!m_TrackPool.TryGet(out SoundIoAudioTrack track))
+            if (!_trackPool.TryGet(out SoundIoAudioTrack track))
             {
                 return -1;
             }
 
-            // Open the output. We currently only support 16-bit signed LE
+            // NOTE: Open the output. We currently only support 16-bit signed LE
```

Comment:
Why have you prepended `NOTE` before the start of each comment? Seems redundant to me...

Reply:
Might be a better idea to do something like that in my pov
```suggestion
            // Open the output.
            // FIXME: We currently only support 16-bit signed LE.
```

Reply:
It's not really a fixme, the entire audio rendering pipeline (service impl and all) only supports 16-bit signed LE at the moment, but I'm not fazed. Just seems pointless to me to add "NOTE" to the start of every comment in the entire codebase.

Reply:
Then we should open an issue add a comment on the class documentation

Reply:
I'm happy with that! :+1: